examples/nlp/text_classification_tf_keras/data.py

```diff
@@ -0,0 +1,130 @@
+import json
```

Comment:
Did you write this data loading code yourself or is it coming from another file? If so, can you put a comment at the top indicating that this is coming from the source code, maybe just slightly refactored for Determined.

Reply:
wrote it myself